### DISCUSSION TITLE
My app is not linked to my account

### DISCUSSION THREAD

**Discussion by Donivanes** [vercel/community#6220] [Account Issue]
### Question

I have an app that was linked to my account and it's not there anymore. The app is still live but i cannot manage it

**Comment by sham601** [comment]
Plz my vercel linked thanks

**Comment by Donivanes** [comment]
@sham601 
This is my vercel username: donivanes
And the app url that It's not linked is: https://tienda-marazuela.vercel.app

**Reply by amyegan** [reply]
Hey @Donivanes. It looks like that project belongs to a Hobby account linked to your GitHub account. If you are looking at a team dashboard, you can switch to your hobby dashboard by using the [team switcher](https://vercel.com/changelog/improved-experience-for-moving-between-your-teams-and-projects) at the top of the page.

**Reply by Donivanes** [reply]
@amyegan this is what i have

<img width="794" alt="Captura de pantalla 2024-03-22 a las 17 25 24" src="https://github.com/vercel/community/assets/22077321/d97ed5d9-e096-47df-979d-bb0b37aa26c8">

**Reply by amyegan** [reply]
Do you have a second Vercel account?

**Reply by Donivanes** [reply]
It should be linked to my GitHub user, I've always signed in with it

**Reply by Donivanes** [reply]
@amyegan any news about this?

**Reply by amyegan** [reply]
I did find the project, but it belongs to another account. I believe that may be a second account belonging to you, based on the name. If you can recall what email address would be associated with your other account, you'll be able to sign in with email to access the project on that account.

Since we can't safely discuss further account details in public, please send an email to registration@vercel.com if you need help logging in to your alternate account. 

https://vercel.com/guides/how-to-get-vercel-support
